Libraries/Map/Map.h

```diff
@@ -1,5 +1,7 @@
 #ifndef Map_h
 #define Map_h
+
+#include "Arduino.h"
```

Comment:
1. All variables must be: named_in_this_way 
    and not: namedInThisOtherWay
2. The members of a class should have at the end _
    For example: my_member_variable_
3. All comments should end with a point.
4. Add a comment before every function in the .h like:
// Returns the number of rows.
uint8_t getRows()
5. Add a comment on every .h file explaining in a brief way what does the class do.
6. better name the functions. Example:
getRows() -> What are you actually receiving? the actual rows? the number of rows?
better: numberOfRows()

Remember to always apply the comment of one file or one part to all the other parts of the code.